deeppavlov/models/squad/README.md

```diff
@@ -0,0 +1,123 @@
+    * **c_r2p**
+    * **spans**
+  * out: 
+    * **ans** - processed answer
+    * **ans_start** - start position of processed answer
+    * **ans_end** - end position of processed answer
+* **squad_vocab_embedder** - builds vocabulary and embedding matrix
+  * in:
+    * **c_tokens**
+    * **q_tokens**
+  * out
+    * **c_tokens_idxs**
+    * **q_tokens_idxs**
+  * fit_on: **c_tokens** and **q_tokens**
+  * level - token or char
+  * emb_folder - path to store pretrained embeddings
+  * emb_url - url to donwload embeddings
+  * save_path - path to save vocabulary and embedding matrix
+  * load_path - path to load vocabulary and embedding matrix
+  * context_limit - maximum length of context in words
+  * question_limit - maximum length of question in words
+  * char_limit - maximum number of chars in token
+* squad_model - model to find answer on question in context  
+  * in: **c_tokens_idxs**, **c_chars_idxs**, **q_tokens_idxs**, **q_chars_idxs**
+  * in_y: **ans_start**, **ans_end**
+  * out:
+    * **ans_start_predicted** - start position of predicted answer
+    * **ans_end_predicted** - end position of predicted answer
+  * word_emb - pretrained word embeddings
+  * char_emb - pretrained char embeddings
+  * context_limit - maximum length of context in words
+  * question_limit - maximum length of question in words
+  * char_limit - maximum number of chars in token
+  * train_char_emb - update char_emb during training or not
+  * char_hidden_size - size of word embedding built on characters
+  * encoder_hidden_size - hidden size of encoder cells 
+  * attention_hidden_size - hidden size to use to compute attention
+  * learning_rate
+  * keep_prob - dropout keep probability
+  * grad_clip - gradient clipping value
+  * save_path
+  * load_path
+* squad_ans_postprocessor - extracts predicted answer from context
+ * in: **ans_start_predicted**, **ans_end_predicted**, **c_raw**, **c_p2r**, **spans**
+ * out: 
+   * **ans_predicted** - text of predicted answer in raw context
+   * **ans_start_predicted** - start position of predicted answer in raw context
+   * **ans_end_predicted** - end position of predicted answer in raw context
+
+## Running model
+**Tensorflow-1.4.0 with GPU support is required** to run this model.
+## Training
+**Warning**: training with default config requires about 10Gb on  GPU. Run following command to train the model:  
+```bash
+python -m deeppavlov.deep train deeppavlov/configs/squad/squad.json
+```
+## Interact mode
+Interact mode provides command line interface to already trained model.
+
+To run model in interact mode run the following command:
+ ```bash
+python -m deeppavlov.deep interact deeppavlov/configs/squad/squad.json
+```
+Model will ask you to type in context and question.
+
+## Pretrained model on SQuAD
+Model is available at the following [[link]](lnsigo.mipt.ru/export/deeppavlov_data/squad_model.tar.gz)
+
+It achieves ~79 F-1 score and ~70 EM on dev set. 
```

Comment:
Position on the leaderboard and comparison with other models would be nice.

Reply:
added link to SQuAD leaderboard